pkg/controller/storagecluster/storagecluster.go

```diff
@@ -437,26 +436,57 @@ func isPreflightComplete(cluster *corev1.StorageCluster) bool {
 	return false
 }
 
+func (c *Controller) createPreFlightStorageNodes(toUpdate *corev1.StorageCluster) error {
+	k8sNodeList := &v1.NodeList{}
+	err := c.client.List(context.TODO(), k8sNodeList)
+	if err != nil {
+		return fmt.Errorf("failed to get cluster nodes for preflight storag node creation: %v", err)
+	}
+
+	// Create StorageNodes to return pre-flight checks used by c.Driver.Validate().
+	for _, node := range k8sNodeList.Items {
+		shouldRun, _, err := c.nodeShouldRunStoragePod(&node, toUpdate)
+		if err != nil {
+			logrus.Infof("Skipping pre-flight storage node entry for %s node. Error: %v", node.Name, err)
```

Comment:
The `Infof` should be an warnf instead as this is not expected. Not error since we aren't failing the operator due to this.